app/assets/stylesheets/finder_frontend.scss

```diff
@@ -56,26 +56,18 @@
   margin-top: govuk-spacing(2);
 }
 
-#finder-frontend {
-  margin-bottom: govuk-spacing(8);
-
-  .signup-content p {
-    margin: govuk-spacing(3) 0;
-    @include govuk-font(19);
+.signup-content p {
+  margin: govuk-spacing(3) 0;
+  @include govuk-font(19);
 
-    @include media(tablet) {
-      margin: govuk-spacing(6) 0;
-      max-width: $two-thirds;
-    }
+  @include media(tablet) {
+    margin: govuk-spacing(6) 0;
+    max-width: $two-thirds;
   }
+}
 
-  legend {
```

Comment:
There are legends on the Q&A pages, which I think is where the overrides are being used. The difference isn't huge though, so may be fine to remove.

## On live
<img width="638" alt="Screen Shot 2019-05-29 at 09 55 42" src="https://user-images.githubusercontent.com/29889908/58543750-f7742500-81f7-11e9-8327-6e74e76fa704.png">

## Heroku
<img width="633" alt="Screen Shot 2019-05-29 at 09 55 28" src="https://user-images.githubusercontent.com/29889908/58543760-fba04280-81f7-11e9-813d-a062e6bea636.png">

Reply:
Ah, great spot. Looks like finder-frontend is increasing the bottom margin from 15px to 20px. As you say, not a huge difference, and also this legend is part of a component and already styled. The override looks like a more general thing, so I wonder if it used to apply to something but doesn't anymore? I think I'm going to take the decision that this is okay to remove. Thanks!